### ISSUE TITLE
Indent rule doesn’t validate variable declarations

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#1800]
The following example has no error:

```js
var a = 1,
b = 2,
c = 3;
```

I would expect the following errors:
```sh
2:0  error  Expected indentation of 4 characters  indent
3:0  error  Expected indentation of 4 characters  indent
```

**Comment by puzrin** [comment]
Make sure, that this style is also valid:

```js
var a = 1
  , b = 2
  , c = 3;
```

**Comment by nzakas** [comment]
@puzrin I'd assume that would only be valid when using two spaces as an indent?

**Comment by puzrin** [comment]
I work only with node's packages, and non 2-spaces style is very rare there. Not enougth experience to give recommendations in general.

https://github.com/nodeca/eslint-plugin-nodeca/blob/master/test/fixtures.yml#L57-L65

See this example, it covers jshint logic. I think, it would be nice to run all those fixtures in bulk via new implementation.

**Comment by gyandeeps** [comment]
I have the code ready for this but i came across an issue that is why I am blocked on this.
Has anyone seen this issue before: https://github.com/estools/estraverse/issues/53
